### ISSUE TITLE
initial velocity in Shlosman & Vitello wind

### ISSUE THREAD

**Issue by jhmatthews** [sirocco-rt/sirocco#482]
In all our wind prescriptions, we need an initial velocity boundary condition. In KWD, we use the sound speed at the launch point, calculated from the temperature profile of the disc. In SV, we use a constant of 6 km/s.

It would be quite nice to have the option of setting the SV v0 to the sound speed for my BLR runs, and generally it makes sense as a boundary condition. This  is easy to do since we just use the inbuilt machinery for KWD, but I wanted to ask how we'd like this to be done for the UI - should it be an advanced option, perhaps?

**Comment by saultyevil** [comment]
I agree. This would be something quite nice to change for my TDE models as well, as using V0 6 km/s seems a bit silly in comparison to the rest of the velocities in the models. 

Setting it as an advanced option, I think, would probably be better, since it's been a good enough value for most models and I don't see much point in adding another standard parameter for the SV wind.

**Comment by jhmatthews** [comment]
Ok, that's what I thought too. I'll wait and see what @Higginbottom and @kslong say, if they agree I will implement it as an advanced option.

**Comment by lazygun37** [comment]
For what it's  worth, this has been an issue for the TDE simulations also. With 6km/s, we're getting crazy high densities in the base, i.e. we're sort of forced to simulate the photosphere...

C

**Comment by kslong** [comment]
For KWD, we set the minimum to a multiple of the sound speed, I believe.   

Anyway, agree it would be a good idea to address this for the SV wind, and as far as I am concerned it does not need to be an advanced parameter.  If all we want is to add a variable to set vzero, or vbase to any constant value that we be fine.  We could just default it to 6, to get the standard SV model.  

Or we could (both for KWD and SV) set up two variables, one to define what type of law we wanted to use (fixed or sound_speed base) for setting the base velocity, and a second parameter (for either the fixed velocity or the sound speed factor).  

Note that I had thought about doing something like this in the (distant) past.  As I recall, the reason it never went further than thinking was I concluded it hardly mattered what you set the base velocity to.  This is because unless you are willing to increase the base velocity by a noticeable fraction of the escape velocity, it did not really change things qualitatively over a very large region of the wind.
